packages/financial-templates-lib/src/price-feed/MedianizerPriceFeed.js

```diff
@@ -28,13 +28,22 @@ class MedianizerPriceFeed extends PriceFeedInterface {
   }
 
   // Takes the median of all of the constituent price feeds' historical prices.
-  getHistoricalPrice(time) {
+  getHistoricalPrice(time, debug = false) {
```

Comment:
nit: maybe call this `verbose` since we're not really using this variable for debugging, but rather to make the script give the user more info.

Reply:
Done